src/components/results/teacher/question-stats-table.tsx

```
@@ -0,0 +1,458 @@
+          break;
+        case "questionText":
+          aValue = a.questionText.toLowerCase();
+          bValue = b.questionText.toLowerCase();
+          break;
+        case "questionType":
+          aValue = a.questionType.toLowerCase();
+          bValue = b.questionType.toLowerCase();
+          break;
+        case "difficulty":
+          // Custom sorting for difficulty
+          const difficultyOrder = { easy: 1, medium: 2, hard: 3, advanced: 4 };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)